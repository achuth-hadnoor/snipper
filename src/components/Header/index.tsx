import React from 'react'

type props = {
    text:string
}

function Header (props:props){
    return(
        <header>
            <h1>{props.text}</h1>
        </header>
    )
}

export default Header;