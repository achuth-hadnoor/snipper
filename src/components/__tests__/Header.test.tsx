import React from 'react';
import { render } from '@testing-library/react';
import Header from '../Header'

test('renders learn react link', () => {
   const { asFragment } = render(<Header text="Hello!"/>); 
   expect(asFragment()).toMatchSnapshot(); // A snapshot is created with which it will compare the HTML generated 
});
