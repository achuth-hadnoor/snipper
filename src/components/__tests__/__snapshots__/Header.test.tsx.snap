// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<DocumentFragment>
  <header>
    <h1>
      Hello!
    </h1>
  </header>
</DocumentFragment>
`;
